script/tool/lib/src/test_command.dart

```diff
@@ -20,7 +19,8 @@ class TestCommand extends PluginCommand {
     argParser.addOption(
       kEnableExperiment,
```

Comment:
What is `kEnableExperiment`? Is it a comma-separated value? or a single experiment? `valueHelp` might be useful here too!

Reply:
Added a link like the one in `flutter`'s help.